.circleci/config.yml

```diff
@@ -56,6 +57,8 @@ jobs:
     executor: rails
     steps:
       - checkout
+      - browser-tools/install-chrome
+      - browser-tools/install-chromedriver
```

Comment:
Confirmed that these are required steps: https://app.circleci.com/pipelines/github/carbonfive/raygun-rails/831/workflows/d53e71e5-7b07-45af-8566-4f995242444f/jobs/3067